guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -4,12 +4,38 @@
 [id="katello-headline-features"]
 == Headline Features
 
-There are no highlights with Katello {KatelloVersion}.
+=== Granular Smart Proxy Content Counts
+
+You can now update smart proxy content counts per lifecycle environment and per content view.
+Smart proxy auto-syncs will now update only the lifecycle environment rather than all counts.
+A substantial performance improvement is expected.
+
+=== Multiple content view environments for hosts and activation keys
+
+Hosts and (new in 4.15) activation keys can now be associated with multiple content view environments. To use this feature, turn on the allow_multiple_content_views Setting.
+Multi-environment hosts and activation keys are displayed in the web UI, but currently can only be changed via Hammer. Use the new --content-view-environments or --content-view-environment-ids options in hammer activation-key create  and hammer activation-key update.
+To display multi-environment hosts in the new All Hosts list page, click 'Manage columns' and add the 'Content view environments' column. The existing content view and lifecycle environment columns will still work, but will only display the host's first content view environment.
+You can also set content view environments from a registered host with subscription-manager environments --set.
+You can now view a list of available content view environments with a new Hammer command, hammer content-view-environment list.
+Multi-environment information is now displayed in hammer host info and hammer activation-key info.
+As part of this change, content view and lifecycle environment must be provided together when creating or updating activation keys. You can no longer assign only a content view or only a lifecycle environment. 
+
+=== Pulpcore upgraded to 3.63
+
+Pulpcore has been upgraded to 3.63, which brings bug fixes and new features that Katello can integrate with.
+As an example, the new Pulp Container now has container image compressed size and OS information available for Katello to show.
+Expect integration with this new data sometime in the near future.
+
+=== New depth option for OSTree repositories
+
+When creating OSTree repositories, users can now determine how much of the OSTree commit history gets synced and published.
+Select the number of commits by entering a number in the new depth field.
 
 [id="katello-upgrade-warnings"]
 == Upgrade Warnings
 
-There are no upgrade warnings with Katello {KatelloVersion}.
+Container content users will want to run 'foreman-maintain advanced procedure run pulpcore-container-handle-image-metadata' to pre-migrate data to avoid a lengthy migration in the future. There will be multiple releases to allow this before it is mandatory.
```

Comment:
```suggestion
Container content users will want to run `foreman-maintain advanced procedure run pulpcore-container-handle-image-metadata` to pre-migrate data to avoid a lengthy migration in the future.
There will be multiple releases to allow this before it is mandatory.
```